### PR TITLE
DOC: Update the Azure Pipelines badge link

### PR DESCRIPTION
**Pull request by thewtex**
The old build was removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ITKModuleTemplate
 =================
 
-[![Build Status](https://dev.azure.com/InsightSoftwareConsortium/ITKModules/_apis/build/status/InsightSoftwareConsortium.ITKModuleTemplate?branchName=master)](https://dev.azure.com/InsightSoftwareConsortium/ITKModules/_build/latest?definitionId=19&branchName=master)
+[![Build Status](https://dev.azure.com/InsightSoftwareConsortium/ITKModules/_apis/build/status/InsightSoftwareConsortium.ITKModuleTemplate%20(1)?branchName=master)](https://dev.azure.com/InsightSoftwareConsortium/ITKModules/_build/latest?definitionId=23&branchName=master
```
Comment (jhlegarreta) [inline README.md]: Missing ending bracket.

Comment (thewtex) [reply README.md]: Good catch @jhlegarreta ! Fixed.

Comment (jhlegarreta) [reply README.md]: Fixed in 8303fa4.

**Comment by thewtex** [comment]
Since I created this PR, Azure decided to resurrect the old build, so this patch is no longer needed.

**Pull request closed without merging**